projects/swimlane/ngx-ui/src/lib/components/drawer/drawer.component.ts

```diff
@@ -110,4 +110,11 @@ export class DrawerComponent implements OnInit, OnDestroy {
   onEscapeKey(): void {
     this.close.emit(true);
   }
+
+  @HostListener('window:resize')
+  onWindowResize(): void {
```

Comment:
Should we throttle this?

Reply:
ya, we could use a set timeout there just would be a tiny delay on the resize

Reply:
after looking into it I am unable to reproduce the issue anymore and am closing this PR